### ISSUE TITLE
code linting readability, increase line length to 100 chars

### ISSUE THREAD

**Issue by mariojmdavid** [indigo-dc/udocker#350]
although PEP8 states 80 char line length, in many places it would profit from 120 char line lenght
examples below
udocker/oci.py lines 24-45

```
        structure["oci-layout"] = \
                self.localrepo.load_json(f_path + "oci-layout")
        structure["index"] = \
                self.localrepo.load_json(f_path + "index.json")
        if not (structure["index"] and structure["oci-layout"]):
            return {}

        for fname in os.listdir(tmp_imagedir + "/blobs"):
            f_path = tmp_imagedir + "/blobs/" + fname
            if FileUtil(f_path).isdir():
                layer_algorithm = fname
                for layer_f in os.listdir(f_path):
                    layer_id = layer_algorithm + ':' + layer_f
                    structure["repolayers"][layer_id] = dict()
                    structure["repolayers"][layer_id]["layer_f"] = \
                            f_path + '/' + layer_f
                    structure["repolayers"][layer_id]["layer_a"] = \
                            layer_algorithm
                    structure["repolayers"][layer_id]["layer_h"] = \
                            layer_f

        return structure
```

would become

```
        structure["oci-layout"] = self.localrepo.load_json(f_path + "oci-layout")
        structure["index"] = self.localrepo.load_json(f_path + "index.json")
        if not (structure["index"] and structure["oci-layout"]):
            return {}

        for fname in os.listdir(tmp_imagedir + "/blobs"):
            f_path = tmp_imagedir + "/blobs/" + fname
            if FileUtil(f_path).isdir():
                layer_algorithm = fname
                for layer_f in os.listdir(f_path):
                    layer_id = layer_algorithm + ':' + layer_f
                    structure["repolayers"][layer_id] = dict()
                    structure["repolayers"][layer_id]["layer_f"] = f_path + '/' + layer_f
                    structure["repolayers"][layer_id]["layer_a"] = layer_algorithm
                    structure["repolayers"][layer_id]["layer_h"] = layer_f

        return structure
```

**Comment by mariojmdavid** [comment]
implemeting 100 chars, should be enough

**Comment by alvarolopez** [comment]
I have to disagree with this issue (see https://pep8.org/#maximum-line-length), but I reckon that 100 chars may be a good compromise.

The correct way to split the lines is to use pharenteses, not backslashes, so the 1st portion or code should be:

        structure["oci-layout"] = self.localrepo.load_json(
            f_path + "oci-layout"
        )
        structure["index"] = self.localrepo.load_json(
            f_path + "index.json"
        )
        if not (structure["index"] and structure["oci-layout"]):
            return {}

        for fname in os.listdir(tmp_imagedir + "/blobs"):
            f_path = tmp_imagedir + "/blobs/" + fname
            if FileUtil(f_path).isdir():
                layer_algorithm = fname
                for layer_f in os.listdir(f_path):
                    layer_id = layer_algorithm + ':' + layer_f
                    structure["repolayers"][layer_id] = {
                        "layer_f": f_path + '/' + layer_f
                        "layer_a": layer_algorithm
                        "layer_h": layer_f
                    }
        return structure

**Comment by mariojmdavid** [comment]
@alvarolopez indeed what I have implemented is 100 chars length, and you are right 120 was too much

**Comment by mariojmdavid** [comment]
issue solved in branch dev-v1.4.0
